task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -198,6 +209,12 @@ spec:
         echo "Prefetched content will be made available"
       fi
 
+      reposdir_path=$(realpath $SOURCE_CODE_DIR)/$REPOSDIR_SRC
+      if [ -d "${reposdir_path}" ]; then
+        chmod -R go+rwX ${reposdir_path}
```

Comment:
Do we need write permission for `go`?

Reply:
We run into multiple permissions issues, and ended up using the same permissions used for the `PREFETCH_INPUT` option. In any case, the permissions are restored to the ones defined for the base image once the build process is done and the volume is unmounted. Let me know if you see an issue here, or any suggested alternative.

Reply:
Would `go+rX` be enough? Does the build ever write into the repos dir?

If the repo files can come from the source directory, it would be ideal if the build did _not_ have the ability to overwrite them (again, for source container accuracy)

Reply:
> Does the build ever write into the repos dir?

Upon doing any yum-related call (e.g. even cleaning the cache), a file called redhat.repo is created under yum.repos.d, so if we want to be able to mount the repos directly there so the user won't have to move them around, then we need to allow writing.

Reply:
Ok, I think we'll have to live with that